### PR TITLE
Major dependency version upgrades

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/package.json [hunk 1]
```diff
@@ -46,31 +46,29 @@
...
+    "prettier": "^1.18.2",
+    "proxyquire": "^2.1.0",
     "qs": "^6.7.0",
     "s3-zip": "^3.0.0",
     "sanitize-filename": "^1.6.1",
+    "sinon": "^7.3.2",
     "tmp": "^0.1.0",
     "uuid": "^3.3.2"
-  },
-  "devDependencies": {
```
Comment (rachaelparris) [inline shared/package.json:62]: Just curious, why did you remove these from dev dependencies?

**Review by rachaelparris — APPROVED** [review]

File: shared/package.json (hunk 1 above)
Comment (ericsorenson) [reply shared/package.json:62]: I combined `dependencies` and `devDependencies` because the only thing that runs in this project is the test framework
